df_auth/viewsets.py

```diff
@@ -52,3 +55,28 @@ def blacklist(self, request, *args, **kwargs):
 class OTPViewSet(ValidationOnlyCreateViewSet):
     serializer_class = OTPObtainSerializer
     permission_classes = (permissions.AllowAny,)
+
+
+class SocialAuth(viewsets.GenericViewSet):
+    serializer_class = SocialTokenObtainSerializer
+
+    def create(self, request, *args, **kwargs):
```

Comment:
this should be DRY, i.e. inherit from the Mixin and you don't need create method at all

Reply:
Done, used the ValidationOnlyCreateViewSet.